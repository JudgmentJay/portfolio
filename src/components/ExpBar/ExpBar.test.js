import React from 'react'
import renderer from 'react-test-renderer'

import { ExpBar } from '../'

const mockData = {
	skill: 'react',
	level: 85
}

it('Renders correctly', () => {
	const tree = renderer
		.create(<ExpBar {...mockData} />)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
