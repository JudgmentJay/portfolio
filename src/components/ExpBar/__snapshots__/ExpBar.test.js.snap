// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="skill"
>
  <span
    className="skillName skillName--small"
  >
    react
  </span>
  <div
    className="bar"
  >
    <div
      className="fill"
      style={
        {
          "width": "25%",
        }
      }
    >
      85
      %
    </div>
  </div>
</div>
`;
